### PR TITLE
Add Hello Microverse Project Files

### PR DESCRIPTION
**Pull request by IsaMarvin**
# :heavy_plus_sign: Added Following Files

- :heavy_check_mark: **a .gitignore file** 
- :heavy_check_mark: **linter files**
- :heavy_check_mark: **Readme file**
- :heavy_check_mark: **license file**
- :heavy_check_mark: **index.html file**
- :heavy_check_mark: **style.css file**
- :heavy_check_mark: **test.md file**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,60 @@
...
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+
+## 📝 License
+
+This project is [MIT](/LICENSE) licensed.
```
Comment (Hamzaoutdoors) [inline README.md:60]: - [ ] Please fix this broken link in your README, while keeping broken links in your documentation is counted as unprofessional, 

you can have something like `[MIT](https://github.com/IsaMarvin/Hello-World/blob/add-hello-world/LICENSE.md)`

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey @IsaMarvin, I am **Hamza** your reviewer for this PR 😸 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README
✔️ Meaningful commit messages 

## Required Changes ♻️

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
**N.A**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @hamzaoutdoors in your question so I can receive the notification ^_^

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,2 +1,60 @@
...
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+
+## 📝 License
+
+This project is [MIT](/https://github.com/IsaMarvin/Hello-World/blob/add-hello-world/LICENSE.md) licensed.
```
Comment (elmar8287) [inline README.md:60]: - [ ] Please, try fixing the request from the previous reviewer, the link is still broken:

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @IsaMarvin.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mateo951 — APPROVED** [review]
### Hi @IsaMarvin, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🔥 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**